README.md

```diff
@@ -252,7 +252,7 @@ Rails.application.configure do |config|
 end
 ```
 
-⚠️ Cache store must implement `#read(key)`, `#write(key, value, **options)` and `#exist?(key)` methods.
+⚠️ Cache store must implement `#read(key)`, `#write(key, value, **options)`, `#delete(key)`, `#clear`, `#keys` and `#exist?(key)` methods.
```

Comment:
Unfortunately [ActiveSupport::Cache::Store](https://github.com/rails/rails/blob/fbe2433be6e052a1acac63c7faf287c52ed3c5ba/activesupport/lib/active_support/cache.rb#L158) does not implement `#keys` (but implements other methods you added), so this requirement will make regular Rails caches incompatible with the gem 🙂 Also, I believe we should not _require_ `delete` and `clear` methods, because the gem itself does not use them